access/types/forms.py

```diff
@@ -862,13 +891,45 @@ def get_randomized_checkbox_attributes(self, i, config, initial, correct,
             ):
                 raise PermissionDenied('Invalid checksum')
 
+        if config.get('randomized'):
+            num_randomized = config.get('randomized')
+            groups = None
+        else:
```

Comment:
For clarity, should here be a comment emphasizing that the else branch always stands for structured-randomized, that is exclusive with randomized case? It may not be obvious for a future reader of the code not seeing this pull request diff, as the file is quite big.

Reply:
Done!